### DISCUSSION TITLE
Cant't make field searchable

### DISCUSSION THREAD

**Discussion by Kikky** [payloadcms/payload#2360] [Feature Requests & Ideas]
Hello!

Here is setting field to be searchable:

`listSearchableFields: ["review"],`

This is field definition in collection:

```
{
      name: "review",
      type: "group",
      index: true,
      fields: [
        {
          name: "header",
          type: "text",
        },
        {
          name: "content",
          type: "textarea",
        },
      ...
      ],
}
```

And I get this error when trying to search

![image](https://user-images.githubusercontent.com/614922/226275920-b07f2655-2602-426f-b8d4-1efab78497f6.png)

## Other Details

Payload version: 1.6.22

**Comment by JessRynkar** [comment]
Hi @Kikky - currently `listSearchableFields` does not support nested groups, if you want to search on these fields they need to be on the top level of your collection config.

This capability is definitely something we plan on implementing in the future - see the related roadmap item [here.](https://github.com/payloadcms/payload/discussions/1570)
